### PR TITLE
Fix date format patient file

### PR DESCRIPTION
**Pull request by eyalvo-matrix**
Fix date format in patient file-> issue screen

### CODE REVIEW

File: interface/patient_file/summary/add_edit_issue.php [hunk 1]
```diff
@@ -224,8 +224,8 @@ function ActiveIssueCodeRecycleFn($thispid2, $ISSUE_TYPES2)
         }
     }
 
-    $form_begin = fixDate($_POST['form_begin'], '');
-    $form_end   = fixDate($_POST['form_end'], '');
+    $form_begin = oeFormatShortDate($_POST['form_begin'], '');
+    $form_end   = oeFormatShortDate($_POST['form_end'], '');
```
Comment (bradymiller) [inline interface/patient_file/summary/add_edit_issue.php]: above should be:
```
    $form_begin = ($_POST['form_begin']) ? DateToYYYYMMDD($_POST['form_begin']) : '';
    $form_end =  ($_POST['form_end']) ? DateToYYYYMMDD($_POST['form_end']) : '';
```

**Comment by bradymiller** [comment]
There's also some js code on this script that uses the dates that I am guessing will need some modification. Here's a function your group created in past for js date stuff: https://github.com/openemr/openemr/blob/master/library/formatting_DateToYYYYMMDD_js.js.php

**Comment by eyalvo-matrix** [comment]
added changed

**Comment by eyalvo-matrix** [comment]
@bradymiller  hi Brady does this need more work ?

**Comment by bradymiller** [comment]
hi @eyalvo-matrix , Code looks good and bringing this in. Thanks for the code improvement! -brady

**Pull request merged**